vault/resource_jwt_auth_backend.go

```diff
@@ -225,11 +237,23 @@ func jwtAuthBackendRead(d *schema.ResourceData, meta interface{}) error {
 		return nil
 	}
 
+	d.Set("type", backend.Type)
+
 	d.Set("accessor", backend.Accessor)
 	for _, configOption := range matchingJwtMountConfigOptions {
 		d.Set(configOption, config.Data[configOption])
 	}
 
+	log.Printf("[DEBUG] Reading jwt auth tune from '%q/tune'", path)
+	rawTune, err := authMountTuneGet(client, "auth/"+path)
+	if err != nil {
+		return fmt.Errorf("error reading tune information from Vault: %s", err)
+	}
+	if err := d.Set("tune", []map[string]interface{}{rawTune}); err != nil {
+		log.Printf("[ERROR] Error when setting tune config from path '%q/tune' to state: %s", path, err)
```

Comment:
Similar question here:
```suggestion
		log.Printf("[ERROR] Error when setting tune config from path %q to state: %s", path+"/tune", err)
```